internal/cmd/request.go

```diff
@@ -8,34 +8,31 @@ import (
 	"github.com/edulinq/autograder/internal/api/core"
 	"github.com/edulinq/autograder/internal/api/server"
 	"github.com/edulinq/autograder/internal/common"
+	"github.com/edulinq/autograder/internal/log"
 	"github.com/edulinq/autograder/internal/util"
 )
 
 type CommonOptions struct {
-	Verbose bool `help:"Use verbose output to show full request/response without specific formatting." default:"false"`
+	Verbose bool `help:"Use verbose output to show the full request/response. Output may not adhere to a valid format/filetype." default:"false"`
 }
 
-func MustHandleCMDRequestAndExit(endpoint string, request any, responseType any) error {
-	return MustHandleCMDRequestAndExitFull(endpoint, request, responseType, CommonOptions{}, nil)
+type CMDPrintHandler func(response core.APIResponse) string
```

Comment:
If this function is not actually printing (which I think it is a good pattern that it does not print),
then you should change the name (since it is not handling printing).
I would also drop the prefix, since it is already in the cmd package.